python/ray/data/dataset.py

```diff
@@ -2456,6 +2458,19 @@ def iter_batches(
                 to select ``numpy.ndarray`` for tensor datasets and
                 ``Dict[str, numpy.ndarray]`` for tabular datasets. Default is "native".
             drop_last: Whether to drop the last batch if it's incomplete.
+            local_shuffle_buffer_size: If non-None, the data will be randomly shuffled
+                using a local in-memory shuffle buffer, and this value will serve as the
+                minimum number of rows that must be in the local in-memory shuffle
+                buffer in order to yield a batch. This is a light-weight alternative to
```

Comment:
Can you mention how the last remainder rows are handled? When there are less than local_shuffle_buffer_size rows, we should let users know if they should expect batches yielded from them.

Reply:
Added a sentence, but the wording feels a bit off... let me know what you think!